source/support/base/Dockerfile

```diff
@@ -9,7 +9,7 @@ FROM augurproject/python2-and-3:latest
 RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - \
 	&& apt-get install -y nodejs
 
-RUN wget --quiet --output-document /usr/local/bin/solc https://github.com/ethereum/solidity/releases/download/v0.4.20/solc-static-linux \
```

Comment:
Unrelated to this PR, but i wonder if we should vendor it or checksum?

Reply:
I have a checksum for it in the monorepo base docker, can copy that over here.